src/query/service/src/pipelines/processors/transforms/transform_cast_schema.rs

```diff
@@ -68,11 +68,15 @@ where Self: Transform
                 }
             })
             .collect();
-        Ok(Transformer::create(input_port, output_port, Self {
-            func_ctx,
-            insert_schema,
-            exprs,
-        }))
+        Ok(ProcessorPtr::create(Transformer::create(
```

Comment:
Move `ProcessorPtr::create` wrap to `Transformer::create` ？

Reply:
The constructor should returns `impl Processor` or `Box<dyn Processor>` so we can wrap it with `ProfileWrapper`.

`ProcessorPtr` is a high-level data struct, it's better to construct it as soon as we need it.